### PR TITLE
Dockerfile for llm and service in docker compose  

### PR DESCRIPTION
**Pull request by Kleczyk**
To run llm you need to download the model from hugingface:
```sh
wget -P ./llm/models https://huggingface.co/TheBloke/Llama-2-7B-GGUF/resolve/main/llama-2-7b.Q3_K_L.gguf
```
smaller model does not work 
### Endpoint to send prompt 
POST 127.0.0.1:9000/v1/completions
Request body

```json
{
  "prompt": "\n\n### Instructions:\nWhat is the capital of France?\n\n### Response:\n",
  "stop": [
    "\n",
    "###"
  ]
}
```
### Successful Response

```json
{
  "id": "string",
  "object": "text_completion",
  "created": 0,
  "model": "string",
  "choices": [
    {
      "text": "string",
      "index": 0,
      "logprobs": {
        "text_offset": [
          0
        ],
        "token_logprobs": [
          0,
          null
        ],
        "tokens": [
          "string"
        ],
        "top_logprobs": [
          {
            "additionalProp1": 0,
            "additionalProp2": 0,
            "additionalProp3": 0
          },
          null
        ]
      },
      "finish_reason": "stop"
    }
  ],
  "usage": {
    "prompt_tokens": 0,
    "completion_tokens": 0,
    "total_tokens": 0
  }
}
```

### Validation Error
```json
{
  "detail": [
    {
      "loc": [
        "string",
        0
      ],
      "msg": "string",
      "type": "string"
    }
  ]
}
```

### CODE REVIEW

File: llm/test/test_llm.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        'prompt': prompt,
+        'stop': stop_signs
+    }
+
+    response = requests.post(url, json=data)
+    
+    assert response.status_code == 200
+    assert 'choices' in response.json()
+    assert len(response.json()['choices']) > 0
+    assert 'text' in response.json()['choices'][0]
```
Comment (pgronkievitz) [inline llm/test/test_llm.py:22]: **suggestion:** removed redundant test and simplified code (it should be a little faster)
```suggestion
	resp_body = response.json()
    assert 'choices' in resp_body
    assert 'text' in resp_body['choices'][0]
```

File: llm/test/test_promt.py
Comment (pgronkievitz) [inline llm/test/test_promt.py:1]: **praise:** this doesn't seem to be proper pytest test and it seems like kinda duplicate of test_llm file

File: llm/test/test_llm.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# test_llm.py
+
+import pytest
+import requests
+
+
+def test_llm_response():
```
Comment (pgronkievitz) [inline llm/test/test_llm.py:7]: **praise:** add [mark it requires running LLM](https://docs.pytest.org/en/latest/how-to/mark.html), custom LLM mark or something like that

**Review by pgronkievitz — CHANGES_REQUESTED** [review]

File: llm/test/test_promt.py
Comment (Kleczyk) [reply llm/test/test_promt.py:1]: this is a quick check of what llm is responding to us sometimes it can be useful

Comment (pgronkievitz) [reply llm/test/test_promt.py:1]: does it have any advantage over running regular test doing exactly the same?

File: README.md [hunk 3]
```diff
@@ -71,12 +71,12 @@ Upon app startup, OpenAI-compatible embedding API will be available at:
...
-#### llamacpp
+#### Download llm model (must have for servis llm to work !!!)
 
-Download models (this can take >1h):
+Download model (size of file 3.6GB ):
 
 ```sh
-wget https://huggingface.co/TheBloke/sheep-duck-llama-2-70B-v1.1-GGUF/resolve/main/sheep-duck-llama-2-70b-v1.1.Q4_K_S.gguf
+curl -o ./llm/models/llama-2-7b.Q3_K_L.gguf -L https://huggingface.co/TheBloke/Llama-2-7B-GGUF/resolve/main/llama-2-7b.Q3_K_L.gguf
 ```
```
Comment (pgronkievitz) [inline README.md:80]: **suggestion (non-blocking):** please add both instructions with curl and wget

**Review by pgronkievitz — APPROVED** [review]
in general lgtm

**Pull request merged**